src/content/pages/knowledgebase/linux/ivpn-on-qubes-os.md

```diff
@@ -0,0 +1,153 @@
+---
+title: IVPN on Qubes OS - IVPN Help
+h1: IVPN on Qubes OS
+url: /knowledgebase/linux/ivpn-on-qubes-os/
+sections:
+    - linux
+sectionTitle: Linux
+layout: help-details
+weight: 8
+---
+# IVPN on Qubes OS
+
+<div markdown="1" class="notice notice--warning">
+This guide was produced using Qubes OS release 4.1.1 (R4.1) and fedora-36 as VM template
+</div>
+
+### Introduction
+
+Qubes OS uses a ProxyVM as an intermediary between other VMs in the system and the outside world. The primary function of a ProxyVM is to handle network traffic for other VMs, allowing those VMs to remain isolated from the internet, untrusted and potentially hostile networks.
+
+In Qubes OS, each VM is assigned a specific role, such as a NetVM (for networking) or an AppVM (for running applications). When an AppVM needs to connect to the internet, it sends its network traffic to the designated NetVM.
+
+Alternatively, an AppVM can send its network traffic directly to a ProxyVM for processing. This configuration, known as a "ProxyVM-based NetVM," allows the ProxyVM to enforce various security policies, such as blocking certain types of traffic or limiting access to specific resources. It also allows the use of a Virtual Private Network (VPN) connection, which can provide additional privacy and security for network traffic.
```

Comment:
Please remove the comma "ProxyVM-based NetVM," -> "ProxyVM-based NetVM" 
Sorry, my fault.

Reply:
fixed